pkgs/development/python-modules/nixpkgs-plugin-update/nixpkgs-plugin-update/src/nixpkgs_plugin_update/__init__.py

```diff
@@ -403,6 +396,7 @@ def add(self, args):
 
             autocommit = not args.no_commit
             if autocommit:
+                assert editor.nixpkgs_repo is not None
```

Comment:
Let's avoid adding asserts to that script. I appreciate your effort to make this typed, but there is no way you can change something in there that affects runtime without breaking everything.

Reply:
We can keep the assertions. I just have PTSD from that script